### DISCUSSION TITLE
Does this tool allow compiling?

### DISCUSSION THREAD

**Discussion by TheBlueJester** [cyrusfirheir/twee3-language-tools#117] [Q&A] [answered]
Does this VS Code plugin allow for the twee files to be complied to HTML or is a secondary tool required for this step?

I've had a look around the extension installed in my VS Code and read the feature list here and haven't seen any mention of compiling to HTML.

Context:
I'm looking at starting a fork of TweeGo that includes updated story format files and has drag and drop compiling and decompiling, a menu/interface and an automated recompiler. But this extension is very popular and it would be wasted time to work on this project if it can be done within VS code already.

**Comment by cyrusfirheir** [comment, marked as answer]
No, this does not include build tooling. For that you still need a compiler like TweeGo or Extwee.

**Reply by TheBlueJester** [reply]
Thanks for the swift and informative reply!

**Reply by TheBlueJester** [reply]
I raise the original question because I wanted to know if I should put in the effort to do this, I've created an updated version of Tweego.

TweeGoMatic
https://github.com/TheBlueJester/TweeGoMatic
